unstructured/ingest/connector/biomed.py

```diff
@@ -170,19 +175,20 @@ def get_file(self):
 
     def write_result(self):
         """Write the structured json result for this doc. result must be json serializable."""
-        output_filename = self._output_filename()
-        output_filename.parent.mkdir(parents=True, exist_ok=True)
-        with open(output_filename, "w") as output_f:
-            output_f.write(json.dumps(self.isd_elems_no_filename, ensure_ascii=False, indent=2))
-        logger.info(f"Wrote {output_filename}")
+        if not self.config.download_only:
```

Comment:
you can save the excess indentation underneath the `if` statement by doing the following instead:

```
    if self.config.download_only:
        return
    outpu_filename = ...
````